osbenchmark/worker_coordinator/worker_coordinator.py

```diff
@@ -1941,7 +2075,7 @@ def clients(self):
 
 # Runs a concrete schedule on one worker client
 # Needs to determine the runners and concrete iterations per client.
-def schedule_for(task, client_index, parameter_source):
+def schedule_for(task_allocation, parameter_source):
     """
     Calculates a client's schedule for a given task.
 
```

Comment:
Same as in ScheduleHandler, docstrings should be updated to replace task with task_allocation and remove client_index